### ISSUE TITLE
[Feature request] Colors from artwork in widgets

### ISSUE THREAD

**Issue by ghost** [timusus/Shuttle#395]
**Shuttle version:**
 
 v2.0.7-beta7
 
**Device, OS:**
 
Samsung Galaxy A6, Android 8.0.0

**Description of request:**
 
It'd be nice to use the colors from the album artwork in the available widgets, in the same way that it is already being applied in-app. Would like to see it as a checkbox when adding a widget.

**Comment by adneal** [comment]
Yeah, I like this idea. I actually worked on it about a year ago, but didn't pushed any changes to Shuttle. It may be worth picking back up now that Shuttle is better at extracting a palette, I think it looks nice.

![screenshot-1501675033252](https://user-images.githubusercontent.com/489364/48676915-464b4e80-eb33-11e8-8eda-03d62d2580ee.jpg)

![screenshot-1501674910733](https://user-images.githubusercontent.com/489364/48676914-421f3100-eb33-11e8-931a-461e3d05936b.jpg)
